exercises/chapter12/test/Main.purs

```diff
@@ -28,6 +30,15 @@ inDir = Path.concat [ "test", "data" ]
 outDir :: FilePath
 outDir = Path.concat [ "test", "data-out" ]
 
+-- If you're behind an http-only proxy, the call to https://reqres.in won't work.
```

Comment:
Thinking something like this for instructions:
> -- If you'd like to run this test offline, you can create this API endpoint locally by installing http-server (npm i -g http-server) and runing it in the "test/data" directory (http-server -p 42524).
-- If you can't reach https://reqres.in due to http-only proxy issues, you may either use the above `http-server` solution, or simply remove the `s` from `https` in the url:
--  reqUrl = http://reqres.in/api/users/1